docs/new-architecture-app-intro.md

```diff
@@ -257,3 +137,371 @@ The `moduleName` has to be the same string used in the `[RCTRootView initWithBri
 // Run pod install with the flags
 RCT_NEW_ARCH_ENABLED=1 pod install
 ```
+
+## Android - Prerequisites
+
+Using the New Architecture on Android has some prerequisites that you need to meet:
+
+1. Using Gradle version >= 7.x
+2. Using Android Gradle Plugin >= 7.x (i.e. the `com.android.tools.build:gradle` dependency)
+
+If you updated to React Native 0.68+, you already meet those prerequisites. If you don't meet them, consider updating those dependencies first.
```

Comment:
should we link to this page in version 0.68 of the doc, to ease the setup of those dependencies?

Reply:
> to this page in version 0.68 of the doc

I think this would be confusing as users will jump between versions, which might be not expected. Users can also Google 'How to update Gradle' and they'll find instructions on how to do it.

If they bump React Native version, they will also be covered.
That's the reason why I streamlined here.